src/components/ConditionalTooltip/ConditionalTooltip.tsx

```diff
@@ -9,7 +9,9 @@ interface Props extends TooltipProps {
 const ConditionalTooltip = ({ show, children, setDisabled, ...rest }: Props) =>
   show ? (
     <Tooltip {...rest}>
-      {children && cloneElement(children, setDisabled ? { isDisabled: setDisabled } : undefined)}
+      <div>
```

Comment:
This fixes an issue where sometimes the tooltip won't show up (this is a known issue with the tooltip in patternfly).